storm-core/src/clj/backtype/storm/converter.clj

```diff
@@ -0,0 +1,200 @@
+(ns backtype.storm.converter
```

Comment:
That seems fine to me.  Lets start with this, and then we can do follow on work later.